token-client/src/test/java/com/sap/cloud/security/xsuaa/client/XsuaaDefaultEndpointsTest.java

```diff
@@ -114,6 +121,6 @@ public void withQueryParametersAndEndingPathDelimiter() {
 	}
 
```

Comment:
do we have a test case which covers the null check -> asserts that the exception is thrown when baseUri is null? If it's not covered would be good to have it here.

Reply:
Done